templates/index.html

```diff
@@ -0,0 +1,196 @@
+            <p class="content">
+              Supervise processes and handle failures gracefully.
+              Monitor and link processes for automatic error recovery and resilient systems.
+            </p>
+          </div>
+        </div>
+      </div>
+
+      <div class="column is-4">
+        <div class="card">
+          <div class="card-content">
+            <div class="has-text-centered mb-4">
+              <span class="icon is-large has-text-danger">
+                <i class="fas fa-3x fa-plug"></i>
+              </span>
+            </div>
+            <h3 class="title is-4 has-text-centered">Pluggable Transports</h3>
+            <p class="content">
+              Generic network transport API supporting TCP, in-memory, and custom backends.
+              Switch between transports without changing application code.
+            </p>
+          </div>
+        </div>
+      </div>
+
+      <div class="column is-4">
+        <div class="card">
+          <div class="card-content">
+            <div class="has-text-centered mb-4">
+              <span class="icon is-large has-text-warning">
+                <i class="fas fa-3x fa-server"></i>
+              </span>
+            </div>
+            <h3 class="title is-4 has-text-centered">Platform Libraries</h3>
+            <p class="content">
+              Rich ecosystem of libraries for common patterns: client/server, async tasks,
+              registries, execution control, and more.
+            </p>
+          </div>
+        </div>
+      </div>
+
+      <div class="column is-4">
+        <div class="card">
+          <div class="card-content">
+            <div class="has-text-centered mb-4">
+              <span class="icon is-large has-text-link">
+                <i class="fas fa-3x fa-code"></i>
+              </span>
+            </div>
+            <h3 class="title is-4 has-text-centered">Type Safe</h3>
+            <p class="content">
+              Leverage Haskell's strong type system for distributed programming.
+              Static closures ensure type safety across network boundaries.
+            </p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<!-- Recent Blog Posts -->
+$if(posts)$
+<section class="section">
+  <div class="container">
+    <h2 class="title is-2 has-text-centered mb-6">Latest Updates</h2>
+    <div class="columns is-multiline">
+      $for(posts)$
+      <div class="column is-4">
+        <div class="card">
+          <div class="card-content">
+            <p class="subtitle is-6 has-text-grey">$date$</p>
+            <h3 class="title is-5">
+              <a href="$url$">$title$</a>
+            </h3>
+          </div>
+        </div>
+      </div>
+      $endfor$
+    </div>
+    <div class="has-text-centered mt-5">
+      <a href="/blog.html" class="button is-primary">View All Posts</a>
+    </div>
+  </div>
+</section>
+$endif$
```

Comment:
I think this section should be removed.

These posts are outdated, which makes the project look dead.